js/aside-script.js

```diff
@@ -287,7 +283,6 @@ function AddCollection(collection,thumbnails)
 
 	list.querySelectorAll("input").forEach(i =>
 		i.addEventListener("focusout",(event) => RenameCollection(i.parentElement.parentElement, event.target.value)));
-	// i.onfocusout=func has issues on some browsers and needs to be implemented with an event listener - https://www.w3schools.com/jsref/event_onfocusout.asp
```

Comment:
Considering we used "i.onclick = X" everywhere else, I worry about the importance of specifying that `i.onfocusout` does not currently work

Reply:
No need to worry, Those are both valid and everyone known approaches, so there's no need to specify why we've chosen one method over another

Reply:
Ok then ! Everything else looks good to me :)